docs/add-new-route-testcase.md

```diff
@@ -0,0 +1,31 @@
+# Add New Route testcase
+
+ ## Prerequisites
+
+User has acount in the system already created by system admin.
+User is log in. 
+User is on a main view.
```

Comment:
I would also add direct link - "main view" might be misleading (and main view might differ for different users in the future).